runtime/errors.go

```diff
@@ -216,3 +217,29 @@ func (e *TransactionParameterTypeNotStorableError) Error() string {
 		e.Type.QualifiedString(),
 	)
 }
+
+// ExtendedParsingCheckingError is a special error which aids in debugging checking problems
```

Comment:
I think it's fine to just name this `ParsingCheckingError`, it's useful in the general case.